lab_1/main.py

```diff
@@ -49,7 +59,13 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+
+    tokens = ['weather', 'sunny', 'man', 'happy', 'and', 'dog', 'happy']
+    freq_dict = {}
+    for i in tokens:
+        if i not in freq_dict:
+            freq_dict[i]=tokens.count(i)
```

Comment:
I guess, you misplaced the function logic. This belongs to the previous function - `calculate_frequencies`.

Here you should get top n words from the dictionary that you build in the `calculate_frequencies`.